core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -1288,6 +1292,23 @@ public boolean isQueryCacheEnabled() {
     public boolean isQueryIgnoreUnknownFunction() {
         return Boolean.parseBoolean(this.getOptional("kylin.query.ignore-unknown-function", "false"));
     }
+    
+    public String getMemCachedHosts() {
+        return getRequired("kylin.cache.memcached.hosts");
+    }
+
+    public boolean isQuerySegmentCacheEnabled() {
+        return Boolean.parseBoolean(getOptional("kylin.query.segment-cache-enabled", "false"));
+    }
+
+    public int getQuerySegmentCacheTimeout() {
```

Comment:
the value is millisecond? then better to add the unit to config name

Reply:
Agree